src/contact_us/index.js

```diff
@@ -0,0 +1,16 @@
+import NavBar from 'main_app/components/NavBar'
+import Footer from 'main_app/components/Footer'
+
+import ContactForm from './components/ContactForm'
+
+function ContactUs () {
+  return (
+    <>
+      <NavBar />
+      <ContactForm />
```

Comment:
Please wrap the main content with an `article` as follows:
```suggestion
      <article>
        <ContactForm />
      </article>
```